xikolo-ios/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -106,6 +122,15 @@ class CourseListViewController : AbstractCourseListViewController {
         }
     }
 
+    @IBAction func unwindLogin(_ segue: UIStoryboardSegue) {
```

Comment:
Somehow yes, but you have to add n empty `unwindLogin` method to every view controller that can trigger the login view. And then you can only update the presenting view controller. Since we have already the `loginStateChanged` notification (for updating the relevant views), let's use that.